docs/docs/shards/index.md

```diff
@@ -559,4 +559,101 @@ A `Float4` type value looks like this: `(Float4 -8.84 38.2 4.7 0.4)`.
     [info] [2022-07-22 22:23:24.076] [T-25152] [logging.cpp::55] [mywire] (9.9, 9.9, 10.1, 9.9)
     ```
 
+### Bytes
+
+Type `Bytes` represents the current memory address of the raw bytes value of the passed data. This type also has a lower case alias `bytes`.
+
+The shard [`(ToBytes)`](https://docs.fragcolor.xyz/shards/General/ToBytes/) extracts and returns the current memory address of the passed input data, as its output.
+
+!!! note
+    Since this is a memory address, it will change everytime you invoke these shards.
+
+=== "Code"
+
+    ```clojure linenums="1"
+    (Bytes "string") (Log)  ;; => current memory address of the "string" data bytes
+    123 (ToBytes) (Log)     ;; => current memory address of the number 123 data bytes
+    ```
+
+=== "Output"
+
+    ```
+    [info] [2022-07-26 17:12:27.878] [T-7536] [logging.cpp::55] [mywire] Bytes: 0x1608eefb9b0 size: 6
+    [info] [2022-07-26 17:12:27.889] [T-7536] [logging.cpp::55] [mywire] Bytes: 0x1608ef78a00 size: 9
+    ```
+
+### Color
+
+Type `Color` represents an RGBA color format and is constructed from four unsigned 8 bit integers (one each for the R, G, B, and A values). This type also has a lower case alias `color`.
```

Comment:
not sure what's the lowercase alias `color` from?

Reply:
It looks like `color` is implemented as a mal type.
```clj
(prn (color 255 256 255 255))
;; => [trace] [2022-07-27 10:04:11.537] [T-26156] [foundation.hpp::300] Destroying wire Audio-DSP-Wire
;; => 255, 0, 255, 255
```
Its also documented under our [/functions/values/#color](https://docs.fragcolor.xyz/functions/values/#color)

Reply:
> not sure what's the lowercase alias `color` from?

It comes from 77cf940f7674eed850ac3cec429659d8cec3c49b.

> This type also has a lower case alias `color`.

It's actually the opposite: mal types are lowercase and for convenience (and backward compatibility with previous examples, a PascalCase was added (see https://github.com/fragcolor-xyz/shards/blob/f6e1f261bc90e540804c6fc95bc8fdad35f047bb/src/mal/SHCore.cpp#L225).

I think we should update this doc to use the lower case version and mention the PascalCase as an alias.

Reply:
Fixed in commit #[190ce35f2cccc0d0714284d4e929d35f5ee6c3f1](https://github.com/fragcolor-xyz/shards/pull/325/commits/190ce35f2cccc0d0714284d4e929d35f5ee6c3f1)

Also, added docs hyperlinks (`.../functions/values/`) for types/values : `seq`, `enum`, and `path`, anticipating their documentation as part of issue #323.
NOTE - `Wire` seems to exist as built-in mal type even though it's named with pascal case.